### ISSUE TITLE
[Visual requirement – Make Code – Green screen] Luminosity ratio is less than 3:1 for focus indicator over the 'Green background & Camera' button under 'Choose a camera' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3618]
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element and border line for edit field if that focus indicator and border line have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Green Screen On" option and select it.
7. Verify that Green Screen setting is getting activated and the controls under "Choose a camera" dialogue. 
9. Verify contrast ratio of focus indicator over the 'Green background & Camera' button.

**Actual Result:**
Luminosity ratio is less than 3:1 for focus indicator over the 'Green background & Camera' button under 'Choose a camera' pop-up and due to this, visual impaired user is unable to see the focus indicator and border line.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for focus indicator over the 'Green background & Camera' button under 'Choose a camera' pop-up, so that visual impaired user could easily see the border of focus.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.11, #Win10-Edge(Chromium)

![WCAG1 4 11_Green Screen_CCA for focus indicator](https://user-images.githubusercontent.com/72871089/98513447-e1441280-228d-11eb-8a4a-da3d1c0eb1c0.jpg)
